transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -164,7 +164,8 @@ void remove(AbstractEpollChannel ch) throws IOException {
     @Override
     protected Queue<Runnable> newTaskQueue(int maxPendingTasks) {
         // This event loop never calls takeTask()
-        return PlatformDependent.newMpscQueue(maxPendingTasks);
+        return maxPendingTasks == Integer.MAX_VALUE ? PlatformDependent.<Runnable>newMpscQueue()
```

Comment:
@Scottmitch why not do this in `PlatformDependent.<Runnable>newMpscQueue(maxPendingTasks);` itself ?